src/expressive.annotations.validate.js

```diff
@@ -524,13 +524,12 @@ var
                             namespacedEvents.push(typeHelper.string.format('{0}.expressive.annotations', event));
                         }
                     });
-                    $(form).find('input, select, textarea').on(namespacedEvents.join(' '), function(event) {
+                    // attach handlers to all inputs that do not have 'ea-bound' class (unless force == true)
+                    $(form).find('input, select, textarea').not(function(idx,el){ return $(el).hasClass('ea-bound') || force;}).on(namespacedEvents.join(' '), function () {

```

Comment:
Ok thanks. It works for a small project I'm working on (didn't notice any sluggishness) so wanted to submit it as an improvement but I only edited it in the browser on github and didn't check out the sample project or anything. I'll go back to the drawing board! Sorry to waste your time.